docs/functions/usage/index.md

```diff
@@ -89,7 +90,7 @@ function App() {
 
   useEffect(() => {
     functions()
-      .httpsCallable('listProducts')()
+      .httpsCallable('listProducts')({ abc: 123 })
```

Comment:
This is changed with `{ abc: 123}` added but no explanation for the change. Could use a comment in the code perhaps?

Reply:
You are right. I removed the addition.